sdk/communication/azure-communication-identity/swagger/README.md

```diff
@@ -56,4 +56,5 @@ service-interface-as-public: true
 custom-types-subpackage: models
 sync-methods: all
 context-client-method-parameter: true
+disable-client-builder: true
```

Comment:
I'd also add a line of `use: '@autorest/java@<version>` where it was the version you used to generate. That way when regenerating you only need to run `autorest` in the command line rather than `autorest --use:<version>`. This also simplifies what is needed to run code generation in general to just have a prerequisite of `npm install -g autorest` as Autorest will handle retrieving the specified Java Autorest version.

Reply:
An example of how this simplifies set-up can be found in Cognitive Search: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/search/azure-search-documents/swagger/README.md#getting-started